vault-minikube/step4.md

```diff
@@ -0,0 +1,51 @@
+Initialize Vault with one key share and one key threshold.
+
+```shell
+kubectl exec vault-0 -- vault operator init -key-shares=1 -key-threshold=1 -format=json > cluster-keys.json
+```{{execute}}
+
+The [`operator
+init`](https://www.vaultproject.io/docs/commands/operator/init.html) command
+generates a master key that it disassembles into key shares `-key-shares=1` and
+then sets the number of key shares required to unseal Vault `-key-threshold=1`.
+These key shares are written to the output as unseal keys in JSON format
+`-format=json`. Here the output is redirected to a file named
+`cluster-keys.json`.
+
+View the unseal key found in `cluster-keys.json`.
```

Comment:
```suggestion
View the unseal key found in `cluster-keys.json`{{open}}.
```

Reply:
Oh that's a good suggestion. I'll make sure I repeat this pattern in this next one as well.